projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1444,39 +1469,75 @@ public function publicize_connections_url( $source = 'calypso-marketing-connecti
 	}
 
 	/**
-	 * Call the WPCOM REST API to get the Publicize shares info.
+	 * Get the Jetpack Social info from the API.
 	 *
-	 * @param string $blog_id The WPCOM blog_id for the current blog.
+	 * @param int $blog_id The WPCOM blog_id for the current blog.
 	 * @return array
 	 */
+	public function get_api_data( $blog_id ) {
+		static $api_data_response = null;
+		$key                      = 'jetpack_social_api_data';
```

Comment:
I've updated the key of this transient, which will invalidate all existing ones. However, it shouldn't be an issue considering this methods makes an API call every time it gets called anyway, and only uses the transient in case of an error.

Reply:
Makes sense. I'm glad you're caching the response for the request too.